### PR TITLE
chore: update library info

### PR DESCRIPTION
**Pull request by sleepdefic1t**
## Proposed changes

Update library information per @kristjank .
These changes will be reflected in external package/library managers.

This PR updates authorship and descriptive content.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (improve a current implementation without adding a new feature or fixing a bug)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build (changes that affect the build system)
- [ ] Docs (documentation only changes)
- [ ] Test (adding missing tests or fixing existing tests)
- [x] Other... Please describe:

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: library.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Ark-Cpp-Client",
-  "keywords": "ark ecosystem, crypto, blockchain, arduino",
-  "description": "Cpp Wrapper for the Ark Ecosystem",
+  "keywords": "ark ecosystem, crypto, blockchain, iot, arduino",
+  "description": "Official Blockchain IoT Cpp Client library provided by the Ark Ecosystem.",
```
[the remark this answers is not in the document]
Comment (sleepdefic1t) [reply library.json]: per @kristjank 
> add "official Blockchain IoT client provided by Ark Ecosystem"

File: library.properties [hunk 2]
```diff
@@ -1,9 +1,8 @@
 name=Ark-Cpp-Client
 version=1.0.0
-author=sleepdefic1t, ciband
-maintainer=sleepdefic1t
-sentence=Ark Ecosystem Cpp Wrapper
-paragraph=A C++/Cpp library with Blockchain/Crypto support for the Ark Ecosystem 
+author=the Ark Ecosystem
+sentence=Official Blockchain IoT Cpp Client library provided by the Ark Ecosystem.
```
[the remark this answers is not in the document]
Comment (sleepdefic1t) [reply library.properties]: per @kristjank 
> add "official Blockchain IoT client provided by Ark Ecosystem"

File: library.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (faustbrian) [reply library.json]: We keep things in line with all other SDKs so the aforementioned description should be used.

File: library.properties (hunk 2 above)
[the remark this answers is not in the document]
Comment (faustbrian) [reply library.properties]: We keep things in line with all other SDKs so the aforementioned description should be used.

File: library.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (sleepdefic1t) [reply library.json]: I'll get with @kristjank and see what he wants to do here. 
Thank you, @faustbrian !

[the remark this answers is not in the document]
Comment (sleepdefic1t) [reply library.json]: Arduino and Particle lbry may require using a different branch or repo.
Descriptions have been updated to match the standard SDK description in the mean time.

File: library.properties (hunk 2 above)
[the remark this answers is not in the document]
Comment (sleepdefic1t) [reply library.properties]: Arduino and Particle lbry may require using a different branch or repo.
Descriptions have been updated to match the standard SDK description in the mean time.

**Pull request merged**